python/tchannel/tornado/dispatch.py

```diff
@@ -115,10 +111,20 @@ def handler(request, response, proxy):
         :param handler:
             A function that gets called with ``Request``, ``Response``, and
             the ``proxy''.
+        :param protocol:
+            Register handler for specific type of format
+            protocol = json/http/thrift/None
+            protocol=None means it registers as raw handle. It deals with raw
+            buffer in the request/response.
```

Comment:
How will the `thrift` protocol work.

Also, not sure you want to call it `protocol`. `scheme` or `arg_scheme`?